hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/txns/crypto/validators/CryptoCreateChecks.java

```diff
@@ -260,11 +260,7 @@ private ResponseCodeEnum validateOnlyAliasProvidedCase(final CryptoCreateTransac
                 return NOT_SUPPORTED;
             }
 
-            if (HederaEvmContractAliases.isMirror(op.getAlias().toByteArray())) {
-                return INVALID_ALIAS_KEY;
-            }
-
-            return isUsedAsAliasCheck(op.getAlias());
+            return INVALID_ALIAS_KEY;
         }
         if (!dynamicProperties.isCryptoCreateWithAliasAndEvmAddressEnabled()) {
```

Comment:
After all changes on HIP-583 the name of the flag can cause confusion. Maybe is good idea to be changed.

Reply:
Do you think having `isCryptoCreateWithAliasEnabled` is better? Or may be you have something different in mind?